scripts/dev.sh

```
@@ -0,0 +1,267 @@
+  wait "$leader" 2>/dev/null || true
+  ! kill -0 "$leader" 2>/dev/null
+  printf '%s|invalid start time\n' "$$" > "$RUNDIR/stale.pid"
+  _stop_pidfile stale
+  [[ ! -e "$RUNDIR/stale.pid" ]]
+  ! FASTI_PORT=0 "$0" --status >/dev/null 2>&1
+  local status_output
+  status_output="$(FASTI_LISTEN=127.0.0.1:18420 "$0" --status)"
+  [[ "$status_output" == *"http://127.0.0.1:18420"* ]]
+  status_output="$(FASTI_LISTEN=127.0.0.1:18420 FASTI_API_URL=http://localhost:18421 "$0" --status)"
+  [[ "$status_output" == *"http://localhost:18421"* ]]
+  ! FASTI_API_URL='http://user:secret@127.0.0.1:18421?token=secret' "$0" --status >/dev/null 2>&1
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)